controllers/storagecluster/storageclasses.go

```diff
@@ -249,14 +267,17 @@ func newCephFilesystemStorageClassConfiguration(initData *ocsv1.StorageCluster)
 }
 
 // newCephBlockPoolStorageClassConfiguration generates configuration options for a Ceph Block Pool StorageClass.
-func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) StorageClassConfiguration {
+func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster, storageClassName string) StorageClassConfiguration {
```

Comment:
Create another function for radosStorageClassConfiguration() that takes StorageClass name as an arg and calls this function for better readability. Do not change existing functions to do multiple things.